docs/src/modules/components/GoogleAnalytics.js

```diff
@@ -139,12 +138,6 @@ function GoogleAnalytics() {
     });
   }, [codeStylingVariant]);
 
-  React.useEffect(() => {
-    window.gtag('set', 'user_properties', {
-      ...getApiPageLayout(),
-    });
-  }, []);
-
```

Comment:
We could keep GA properties if the default layout is changed?

I would want to see who is going for:

<img src="https://github.com/mui/material-ui/assets/3165635/5de13855-8af0-4000-982d-82a76ac1ecff" width="180">

if the adoption is low, e.g. 5% I would be in favor of removing, reducing mental overhead.

Reply:
> We could keep GA properties if the default layout is changed?

The current GA behavior is to store layout as user settings. which was a requirement because for the same user we wanted to track the randomly assigned layout and the chosen one.

Since this requirement does not exist now, I propose to move the GA to a system where we track modification instead of the saved state. Each time a user click on one of those button, it send an GA event.

> I would want to see who is going for collapsed

Me too and that's par of the reason. I use it when I'm searching for a specific props. And this toggle of collapsed/expanded is for now not visible in our data collected because we only store one state per user. It should be visible in the new one since we will listen at all clicks

Reply:
> Since this requirement does not exist now, I propose to move the GA to a system where we track modification instead of the saved state. Each time a user click on one of those button, it send an GA event.

I don't know if we can rely on this data. I have tried a bit https://analytics.google.com/analytics/web/?authuser=1#/analysis/p353089763/edit/6jil-1NpQv-a3sMCL6YFlw

<img width="847" alt="SCR-20240319-bjbs" src="https://github.com/mui/material-ui/assets/3165635/32d465b8-b5ba-4622-80e3-e9bd18986a30">

but how to judge people trying one option out and revert to the ones who stays on it?